tools/mock_node/src/setup.rs

```diff
@@ -340,8 +305,8 @@ pub fn setup_mock_node(
 }
 
 #[cfg(test)]
-mod test {
-    use crate::setup::{setup_mock_node, MockNetworkMode};
+mod tests  {
```

Comment:
@marcelo-gonzalez the idiomatic name here is `tests`, and one generally doesn't have to type it manually. There's `tmod` built-in snippet in both Clion and rust-analyzer which auto-comples to the whole stanza with cfg

Reply:
cc @mzhangmzz